### PR TITLE
Fix neutronium replication crash, add item/fluid buffer

### PR DESCRIPTION
**Pull request by hjae78**

### CODE REVIEW

File: src/main/java/gregicadditions/machines/GATileEntities.java [hunk 1]
```diff
@@ -720,7 +725,9 @@ public static void init() {
             OUTPUT_HATCH_FILTERED.add(GregTechAPI.registerMetaTileEntity(id++, new MetaTileEntityOutputFilteredHatch(location("fluid_hatch.export_filtered." + GTValues.VN[i].toLowerCase()), i)));
         }
 
-
+        BUFFER[0] = GregTechAPI.registerMetaTileEntity(id++, new TileEntityBuffer(location("buffer.lv"), 3));
```
Comment (huneau) [inline src/main/java/gregicadditions/machines/GATileEntities.java:728]: can you force id plz? I'm regrating that increment id
3230
3231
3232 should be good

File: src/main/java/gregicadditions/recipes/GARecipeAddition.java [hunk 2]
```diff
@@ -2554,7 +2555,7 @@ else if (!GAConfig.GT6.BendingFoilsAutomatic || !GAConfig.GT6.BendingCylinders)
                 .buildAndRegister();
         ELECTROLYZER_RECIPES.recipeBuilder()
                 .fluidInputs(XenoauricFluoroantimonicAcid.getFluid(29000))
-                .fluidOutputs(Gold.getFluid(1000))
+                .outputs(OreDictUnifier.get(dust, Gold))
                 .fluidOutputs(Xenon.getFluid(2000))
                 .fluidOutputs(Antimony.getFluid(4000))
```
Comment (huneau) [inline src/main/java/gregicadditions/recipes/GARecipeAddition.java:2560]: antimony dust output ?

File: src/main/java/gregicadditions/recipes/GARecipeAddition.java [hunk 3]
```diff
@@ -2947,6 +2948,11 @@ else if (!GAConfig.GT6.BendingFoilsAutomatic || !GAConfig.GT6.BendingCylinders)
         LASER_ENGRAVER_RECIPES.recipeBuilder().duration(50).EUt(30720).inputs(WAFER_AMERICIUM.getStackForm()).notConsumable(OrePrefix.craftingLens, MarkerMaterials.Color.Blue).outputs(MetaItems.POWER_INTEGRATED_CIRCUIT_WAFER.getStackForm(12)).buildAndRegister();
         LASER_ENGRAVER_RECIPES.recipeBuilder().duration(25).EUt(122880).inputs(WAFER_NEUTRONIUM.getStackForm()).notConsumable(OrePrefix.craftingLens, MarkerMaterials.Color.Blue).outputs(MetaItems.POWER_INTEGRATED_CIRCUIT_WAFER.getStackForm(16)).buildAndRegister();
 
+        ModHandler.addShapedRecipe("buffer_lv", GATileEntities.BUFFER[0].getStackForm(), " G ", " H ", " C ", 'G', GAMetaBlocks.TRANSPARENT_CASING.getItemVariant(GATransparentCasing.CasingType.REINFORCED_GLASS, 1), 'H', MetaTileEntities.HULL[GTValues.LV].getStackForm(), 'C', OreDictNames.chestWood);
```
Comment (huneau) [inline src/main/java/gregicadditions/recipes/GARecipeAddition.java:2951]: can you add metatile recipe into MachineRecipe.java (forget the exact name of this file)

File: src/main/java/gregicadditions/item/GAMetaItems.java [hunk 4]
```diff
@@ -207,6 +207,10 @@
 
     public static MetaItem<?>.MetaValueItem UNSTABLE_STAR;
 
+    public static MetaItem<?>.MetaValueItem HYPERIUM_ESSENCE;
```
Comment (huneau) [inline src/main/java/gregicadditions/item/GAMetaItems.java:210]: I think there is a MetaItem file for MA

File: src/main/java/gregicadditions/recipes/GeneratorFuels.java [hunk 5]
```diff
@@ -114,9 +114,9 @@ public static void init() {
 
         //rocket fuel
         registerRocketFuel(RocketFuelH8N4C2O4.getFluid(6), 10, GTValues.EV);
-        registerRocketFuel(RocketFuelCN3H7O3.getFluid(12), 10, GTValues.EV);
+        registerRocketFuel(RocketFuelCN3H7O3.getFluid(10), 10, GTValues.EV);
```
Comment (huneau) [inline src/main/java/gregicadditions/recipes/GeneratorFuels.java:117]: we should do some math, it seems inefficient

**Review by huneau — CHANGES_REQUESTED** [review]

File: src/main/java/gregicadditions/recipes/GeneratorFuels.java (hunk 5 above)
Comment (hjae78) [reply src/main/java/gregicadditions/recipes/GeneratorFuels.java:117]: I did full math on both these fuels, 100 rp-1 rocket fuel takes 127800 EU to produce and 50 CN3H7O3 takes 25725 EU to produce, I'm not sure what kind of efficiency you want for them

Comment (hjae78) [reply src/main/java/gregicadditions/recipes/GeneratorFuels.java:117]: That is all recipes at no overclock

File: src/main/java/gregicadditions/item/GAMetaItems.java (hunk 4 above)
Comment (hjae78) [reply src/main/java/gregicadditions/item/GAMetaItems.java:210]: No, all MA integration items are their own individual items, no MetaItem available

File: src/main/java/gregicadditions/recipes/GeneratorFuels.java [hunk 6]
```diff
@@ -114,9 +114,9 @@ public static void init() {
...
         registerRocketFuel(DenseHydrazineFuelMixture.getFluid(18), 10, GTValues.EV);
-        registerRocketFuel(RP1RocketFuel.getFluid(24), 10, GTValues.EV);
+        registerRocketFuel(RP1RocketFuel.getFluid(12), 10, GTValues.EV);
```
Comment (huneau) [inline src/main/java/gregicadditions/recipes/GeneratorFuels.java:119]: can you change to 
RocketFuelH8N4C2O4 => 3
RocketFuelCN3H7O3 => 6
DenseHydrazineFuelMixture => 9

**Review by huneau — CHANGES_REQUESTED** [review]

**Pull request merged**